src/js/modal.js

```diff
@@ -777,6 +778,34 @@ $(document).ready(function() {
 		document.body.appendChild(modal)
 	}
 
+	//simple modal with text, that user has to click continue to proceed.
+     modals.noticeModal = function(title, desc, method) {
+ 	      var confirmBtn = el('button#continue.btn', ['continue'])
+ 	      var modal = el('div#modalNotice.modal.notice', [
+ 	              	el('div.{overflow-x: visible};', [
+ 	                el("div.modal-header-title", [title]),
+ 	                el("div#bottom-divider.modal-divider"),
+ 	                el("div", [desc]),
+ 	                confirmBtn
+ 	            ])
+ 	          ])
+
+ 	    //removes escape event listeners so user can only press continue to proceed.
+ 	   	var old_el = document.body
```

Comment:
That's a pretty smart idea. Maybe add a note somewhere that calling this function probably breaks all subsequent modals

Reply:
Or you could add an exclusion on the ESC keydown listener